apps/demo/src/scss/mixins/_cart-total.scss

```diff
@@ -1,4 +1,5 @@
-@import "../typography/mixins";
```

Comment:
I think this mixin can be killed. It's not used anywhere and doesn't really make sense. We have a better implementation already in place using `@Input() emphasize: boolean;` in `cart-totals-item`.

Reply:
@xelaint agreed removed in c36294e69967775496f33613cc1d556e30ff9446